### ISSUE TITLE
Compo ranking drag&drop doesn't work correctly (due to tie?)

### ISSUE THREAD

**Issue by sagamusix** [demozoo/demozoo#290]
Take the demo compo from https://demozoo.org/parties/3023/ which has two 6th places and an 8th place. I tried dragging the 8th place entry to the end of the table, which works visually but as soon as I submit the form, it ends up between the two 6th places again.

**Comment by DannyCork** [comment]
I was *unable* to to reproduce , see gif
![jun-27-2017 21-53-09](https://user-images.githubusercontent.com/3512672/27609560-1e587114-5b83-11e7-9d1f-9c31da3bdbef.gif)

**Comment by sagamusix** [comment]
Yes, what you did is not what I described. Read again. When I reported the issue, the 8th place was *not* last, and I tried dragging it to the last place, which failed. There were other ways to do it by dragging the *other* entries instead, so someone went ahead a fixed the compos so it's no longer possible to directly reproduce this in the state it used to be.
